app/models/category.rb

```diff
@@ -0,0 +1,7 @@
+class Category < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :expense_categories, dependent: :destroy
+  has_many :expenses, through: :expense_categories, dependent: :destroy
```

Comment:
[OPTIONAL] I would recommend removing line 3 since the Category has many expenses through expense_categories.